rfcs/0050-roadmaps.md

```diff
@@ -0,0 +1,320 @@
+- Feature Name: roadmaps
+- Start Date: 2022-01-07
+- RFC PR: [apache/tvm-rfcs#0050](https://github.com/apache/tvm-rfcs/pull/0050)
+- GitHub Issue: [apache/tvm#0000](https://github.com/apache/tvm/issues/0000)
+- pre-RFC: https://discuss.tvm.apache.org/t/pre-rfc-tvm-roadmap/11171
+- pre-RFC: https://discuss.tvm.apache.org/t/pre-rfc-roadmap-rfcs/11711
+- Co-Authors: Denise Kutnick (@denise-k), Andrew Reusch (@areusch)
+
+NOTE(areusch): This RFC is a combination of the two above pre-RFCs, and limited copy-editing has
+been done to preserve the flow of the document.
+
+## Summary
+
+This RFC proposes to add product roadmaps to TVM.
+
+Roadmaps should be seen as a way of unifying the planning process of TVM, all the way from ideas to
+PR merges. The roadmaps discussed in this pre-RFC are intentionally designed to integrate with TVM’s
+existing planning tools (e.g. Github tracking issues, RFCs), while adding an additional space for
+sharing and collaboration earlier in the R&D process.
+
+This proposal includes two categories of roadmaps:
+
+- **TVM Global Roadmap**: This roadmap aims to provide a holistic view of all components within TVM,
```

Comment:
1. Is the global roadmap any different than the sum of the component roadmaps?
2. How do we keep the global roadmap from falling into disrepair as teams update their own component roadmaps? a.k.a. who owns / drives it?

Reply:
@driazati, to answer your questions:
1. The global roadmap is intended to be coarser-grained whereas the component roadmaps are intended to be finer-grained. If the tracking does ever overlap, both sets of roadmaps would be using the same base issues as roadmap items. For example, the TVM Global Roadmap and the TVM Automation Roadmap may both contain the [AutoTIR tracking issue](https://github.com/apache/tvm/issues/8473).
2. In general, upon introduction of any new roadmap, we are requesting that there is at least one maintainer who volunteers to keep the roadmap up to date. For the TVM Global Roadmap specifically, we anticipate that this roadmap will be maintained by several active contributors within the TVM Community, and will address this further in the TVM Global Roadmap RFC.